tools/tasks/tslint.ts

```diff
@@ -11,7 +11,9 @@ export = function tslint(gulp, plugins) {
     ];
 
     return gulp.src(src)
-      .pipe(plugins.tslint())
+      .pipe(plugins.tslint({
+        rulesDirectory: NG2LINT_RULES
```

Comment:
It could still be added to `tslin.json` so if running `tslint` in a shell it will work as well.